modules/withRouter.js

```diff
@@ -1,18 +1,28 @@
 import React from 'react'
 import hoistStatics from 'hoist-non-react-statics'
 import { routerShape } from './PropTypes'
+import warning from './routerWarning'
+
 
 function getDisplayName(WrappedComponent) {
   return WrappedComponent.displayName || WrappedComponent.name || 'Component'
 }
 
-export default function withRouter(WrappedComponent) {
+export default function withRouter(WrappedComponent, options) {
```

Comment:
Huh? You can just do `withRouter(WrappedComponent, { withRef = false })`. But the default is `undefined`, so it's falsey and would work all the same.